index.js

```diff
@@ -14,6 +14,15 @@ function Plugin(options) {
   if (this.options.logTime === undefined) {
     this.options.logTime = DEFAULT_LOG_TIME;
   }
+
+  if (typeof this.options.indent === 'undefined') { this.options.indent = null; }
+  if (this.options.indent !== null) {
+    if (typeof this.options.indent === 'boolean' && this.options.indent) {
+      this.options.indent = this.options.indent ? 2 : null;
+    } else if (typeof this.options.indent !== 'string' && typeof this.options.indent !== 'number') {
+      throw new Error('Indent should be of type boolean, number, or string, was ' + typeof this.options.indent);
+    }
+  }
```

Comment:
No point in doing this. Let's JSON.stringify take care of it. In docs, we can mention that the value is passed as is to JSON.stringify and may be add a link to JSON.stringify docs.